querybook/webapp/components/DataDocLeftSidebar/DataDocLeftSidebar.tsx

```diff
@@ -57,13 +57,11 @@ export const DataDocLeftSidebar: React.FunctionComponent<IProps> = ({
         }, [])
     );
 
-    useEffect(
-        () => () => {
-            clearSidebarTableId();
-            setContentState('default');
-        },
-        []
-    );
+    useEffect(() => {
+        clearSidebarTableId();
+        setContentState('default');
+    }, []);
```

Comment:
why this change?

Reply:
Opportunistic cleanup - removes the extra arrow function `() => () => {`

Reply:
`()=>()=>{}`  will do the cleanup when the components unmounts, while `()=>{}` cleans up when the components mounts.

just to make sure if the ui behavior still looks good